protobuf/fmp/fmp.proto

```diff
@@ -9,6 +9,9 @@ option go_package = "fmp_pb";
 message Devices {
     map<string, Device> devices = 1;
     int64 timestamp = 2; //The moment that this metadata was generated
+
+    //Extra information describing locality positions
+    map<string, Locality> localities = 3;
```

Comment:
Doesn't `devices` already has the locality?

Reply:
@bpeng The devices has the `locality` string. i.e. the name of the locality.

This is adding a map from that name to the location.